src/App.js

```diff
@@ -1,11 +1,163 @@
 import React from 'react';
 import './App.css';
+import {useState} from "react";
+import { useForm } from 'react-hook-form';
+
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
+
+  const [count,setCount] = useState(0);
+  const [countTwo,setCountTwo] = useState(0);
+  const [countThree,setCountThree] = useState(0);
+  const [countFour,setCountFour] = useState(0);
+
+  const resetCounts = () => {
+    setCount ( 0), setCountTwo(0), setCountThree(0), setCountFour(0);
+  }
+  const [succes, setSucces] = useState(false);
+  const { register, handleSubmit, formState: {errors}, } = useForm();
+  //function onSubmit takes the data from the form and logs it
+  const onSubmit = data => console.log(data);
+
+  console.log(errors);
+
+
+    return (
+
+  <section>
+    <h1>Fruitmand bezorgservice</h1>
+    <h2>Je mag toch niet naar buiten</h2>
+
+    <section className={"form-section"}>
+      <form className={"form-one"}>
+
+        <div className={"fruit-items"}>
+          <label htmlFor="strawberry"> Aardbeien</label>
+          <button
+              name="strawberry"
+              id="strawberry"
+              value={count}
+              onClick={()=> setCount(count + 1)}> +</button>
+            <p>{count}</p>
+          <button onClick={()=> setCount(count - 1)}>-</button>
+        </div>
+
+        <div className={"fruit-items"}>
+          <label htmlFor="bananas"> Bananen</label>
+          <button
+              name="bananas"
+              id="bananas"
+              value={countTwo}
+              onClick={()=> setCountTwo(countTwo + 1)}>
+            +
+          </button>
+            <p>{countTwo}</p>
+          <button onClick={()=> setCountTwo(countTwo - 1)}>
+            -
+          </button>
+        </div>
+
+        <div className={"fruit-items"}>
+          <label htmlFor="apples"> Appels</label>
+          <button
+              name="apples"
+              id="apples"
+              value={countThree}
+              onClick={()=> setCountThree(countThree + 1)}> +</button>
+            <p>{countThree}</p>
+          <button onClick={()=> setCountThree(countThree - 1)}>-</button>
+        </div>
+
+        <div className={"fruit-items"}>
+          <label htmlFor="kiwis"> Kiwi's</label>
+          <button
+              name="kiwis"
+              id="kiwis"
+              value={countFour}
+              onClick={()=> setCountFour(countFour + 1)}> +</button>
+            <p>{countFour}</p>
+          <button onClick={()=> setCountFour(countFour - 1)}>-</button>
+        </div>
+
+        <button
+            name="reset"
+            id= "reset"
+          onClick={()=> {resetCounts()}}
```

Comment:
Als de functie geen parameters krijgt, dan kun je die er 'direct' in zetten: `onClick={resetCounts}`